### PR TITLE
Run more apps

### PR DESCRIPTION
**Pull request by shijinabraham-leapyear**
- Had to use name rddops instead of rdd-ops
- spark-submit  --packages com.databricks:spark-csv_2.10:1.2.0 for dataframe

### CODE REVIEW

**Comment by robinbb** [comment]
@shijinabraham-leapyear Please resolve merge conflicts.

**Comment by shijinabraham-leapyear** [comment]
Done

File: .travis.yml [hunk 1]
```diff
@@ -24,13 +24,20 @@ before_install:
...
+  
 script:
   - nix-shell --run "./build.sh --fast --test"
   - nix-shell --run "stack exec -- sparkle package sparkle-example-hello"
   - nix-shell --run 'spark-submit $(stack exec -- which sparkle-example-hello).jar'
-  - ./build.sh
+  - nix-shell --run "stack exec -- sparkle package sparkle-example-argv"
+  - nix-shell --run 'spark-submit $(stack exec -- which sparkle-example-argv).jar'
+  - nix-shell --run "stack exec -- sparkle package sparkle-example-dataframe"
+  - nix-shell --run 'spark-submit  --packages com.databricks:spark-csv_2.10:1.2.0 $(stack exec -- which sparkle-example-dataframe).jar'
```
Comment (robinbb) [inline .travis.yml]: Good catch with this spark-csv package!

**Comment by shijinabraham-leapyear** [comment]
Sorry. I have added ./build.sh back.

File: .travis.yml [hunk 2]
```diff
@@ -24,12 +24,20 @@ before_install:
 
 install:
   - nix-shell --run "./clean.sh"
-  - nix-shell --run "./build.sh --fast --test --only-snapshot"  
-
+  - nix-shell --run "./build.sh --fast --test --only-snapshot"
+
```
Comment (robinbb) [inline .travis.yml]: Remove spaces on blank line.

Comment (shijinabraham-leapyear) [inline .travis.yml]: done

**Comment by robinbb** [comment]
@shijinabraham-leapyear The leapyear-patchset branch is maintained as a set of patches "on top of" patches from upstream. So, all merges must be "rebase and merge"-types. Your changes cannot be rebased by Github automatically. Please reformulate this change as a single commit on top of leapyear-patchset, and then force-push to your run_more_apps branch. Changes look good!

**Comment by shijinabraham-leapyear** [comment]
Rebaseing and force push has made this branch in a weird state. I am not able to repair it .Closing this PR and raising https://github.com/LeapYear/sparkle/pull/66 instead

**Pull request closed without merging**